contracts/settling_game/L01_Settling.cairo

```diff
@@ -63,6 +65,26 @@ func settle{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(t
     # TODO: TimeStamp - current Hardcoded
     IS01_Settling.set_time_staked(settle_state_address, token_id, block_timestamp)
 
+    # updated settled realms counter
+    let (realms_settled) = IS01_Settling.get_total_realms_settled(contract_address=settle_state_address)
+    
+    IS01_Settling.set_total_realms_settled(settle_state_address, realms_settled + 1)
+
+    # update wonders
+    let (realms_data : RealmData) = realms_IERC721.fetch_realm_data(
+        contract_address=realms_address, token_id=token_id)
+
+    if realms_data.wonder == 1:
+        let (wonders_state_address) = IModuleController.get_module_address(
+        contract_address=controller, module_id=9)
+
+        # update info for epoch
```

Comment:
I'd argue this (and below, in `unsettle`) should be done in the Wonder module. For one, instead of 3 external calls you'll have only one and ideally to the "logic" part of the module, not the "state".